lang/strings_chinese_traditional.txt

```diff
@@ -1230,6 +1230,7 @@ $s_month_october = '10 月';
 $s_month_november = '11 月';
 $s_month_december = '12 月';
 $s_timeline_issue_created = '<span class="username">%1$s</span> 已建立問題 <span class="issue_id">%2$s</span>';
+$s_timeline_issue_file_added = '<span class="username">%1$s</span> 已新增檔案於議題 <span class="issue_id">%2$s</span>';
```

Comment:
As mentioned by @atrol, translations should be made in TranslateWiki after the PR is merged. This should not be part of the change